app/Main.hs

```diff
@@ -7,4 +7,11 @@ import qualified Data.ByteString.Lazy as B (getContents)
 main :: IO ()
 main = do
   json <- B.getContents
-  print (extractViewerCount json)
+  print $ humanViewerCount $ extractViewerCount json
+
+humanViewerCount :: Maybe Int -> [Char]
+humanViewerCount (Just i)
+  | i > 1     = "There are " ++ (show i) ++ " viewers."
+  | i == 1    = "There is only "  ++ (show i) ++ " viewer."
+  | otherwise = "There are no viewers!"
```

Comment:
Tu peux faire une version courte et francophone s'il-te-plait :

```haskell
humanViewerCount :: Maybe Int -> String
humanViewerCount (Just i)
  | i > 1 = show i <> " specateurs"
  | i == 1 = show i ++ " spectateur"
  | otherwise = offline
humanViewerCount Nothing = offline

offline :: String
offline = "offline"

Reply:
👍 ✅